src/hammer-vlsi/hammer_driver.py

```diff
@@ -148,6 +153,80 @@ def update_tool_configs(self) -> None:
         tools = reduce(lambda a, b: a + b, list(self.tool_configs.values()))
         self.database.update_tools(tools)
 
+    def load_formal_tool(self, run_dir: str = "") -> bool:
+        """
+        Load the place and route tool based on the given database.
+
+        :param run_dir: Directory to use for the tool run_dir. Defaults to the run_dir passed in the HammerDriver
+                        constructor.
+        """
+        if run_dir == "":
+            # Use the synthesis directory for formal tools
+            run_dir = os.path.join(self.obj_dir, "syn-rundir")
```

Comment:
Why not make a new directory?

Reply:
The formality script is packaged in the DC script tarball.